cluster/network-plugins/cni/to_docker/bin/c2d

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+INPUT=/tmp/c2d-$$-in
+RESULT=/tmp/c2d-$$-out
+LOG=/tmp/c2d-$$-log
+cat > "${INPUT}"
+"${0}-inner" "${INPUT}" "${RESULT}" &> "${LOG}"
+RC=$?
+if [ "${RC}" == "0" ]; then
+    cat "${RESULT}"
+else
+    echo '{'
```

Comment:
if you wrote this as a here-doc I think it would read better:
```
cat << __EOF
{
    "cniVersion": "0.1.0",
    "code": "${RC},"
    "msg": "${0}-inner returned ${RC}",
}
__EOF
```